optimism.tokenlist.json

```diff
@@ -707,6 +707,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 42,
+      "address": "0x002be8a5961e0f352092d6693133a6944b7846ba",
+      "name": "Perpetual",
+      "symbol": "PERP",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/PERP.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x51B2bBeb6808Bd81be2a163Bce7374F3EC334E82",
```

Comment:
Not sure why you created your own contract here if it's based entirely on `L2StandardERC20`. You can just use the Token factory here https://kovan-optimistic.etherscan.io/address/0x4200000000000000000000000000000000000012#writeContract and call `createStandardL2Token`

Reply:
oh, we didn't realize it. Will update it later.

Reply:
new address is `0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb`
create from token factory
- https://kovan-optimistic.etherscan.io/address/0x50c5725949a6f0c72e6c4a641f24049a917db0cb#internaltx